tools/wheels/upload_wheels.sh

```diff
@@ -0,0 +1,62 @@
+# Copied from numpy version
+# https://github.com/numpy/numpy/blob/main/tools/wheels/upload_wheels.sh
+
+set_travis_vars() {
+    # Set env vars
+    echo "TRAVIS_EVENT_TYPE is $TRAVIS_EVENT_TYPE"
+    echo "TRAVIS_TAG is $TRAVIS_TAG"
+    if [[ "$TRAVIS_EVENT_TYPE" == "push" && "$TRAVIS_TAG" == v* ]]; then
+      IS_PUSH="true"
+    else
+      IS_PUSH="false"
+    fi
+    if [[ "$TRAVIS_EVENT_TYPE" == "cron" ]]; then
+      IS_SCHEDULE_DISPATCH="true"
+    elif [[ "$TRAVIS_EVENT_TYPE" == "api" ]]; then
+      # Manual CI run, so upload
+      IS_SCHEDULE_DISPATCH="true"
+    else
+      IS_SCHEDULE_DISPATCH="false"
+    fi
+}
+set_upload_vars() {
+    echo "IS_PUSH is $IS_PUSH"
+    echo "IS_SCHEDULE_DISPATCH is $IS_SCHEDULE_DISPATCH"
+    if [[ "$IS_PUSH" == "true" ]]; then
+        echo push and tag event
+        export ANACONDA_ORG="multibuild-wheels-staging"
+        export TOKEN="$SCIPY_STAGING_UPLOAD_TOKEN"
+        export ANACONDA_UPLOAD="true"
+    elif [[ "$IS_SCHEDULE_DISPATCH" == "true" ]]; then
+        echo scheduled or dispatched event
+        export ANACONDA_ORG="scipy-wheels-nightly"
+        export TOKEN="$SCIPY_NIGHTLY_UPLOAD_TOKEN"
+        export ANACONDA_UPLOAD="true"
+    else
+        echo non-dispatch event
+        export ANACONDA_UPLOAD="false"
+    fi
+}
+upload_wheels() {
+    echo ${PWD}
+    if [[ ${ANACONDA_UPLOAD} == true ]]; then
+        if [ -z ${TOKEN} ]; then
+            echo no token set, not uploading
+        else
+            python -m pip install \
+            git+https://github.com/Anaconda-Platform/anaconda-client.git@be1e14936a8e947da94d026c990715f0596d7043
```

Comment:
It would be nice if this could use PyPI-wheels instead of a git checkout. xref Anaconda-Platform/anaconda-client#540